generators/php/codegen/src/ast/Class.ts

```diff
@@ -59,7 +63,11 @@ export class Class extends AstNode {
         }
         this.writeComment(writer);
         writer.writeLine(`class ${this.name}`);
-        writer.writeLine("{");
+        if (this.parentClassReference != null) {
+            writer.write(" extends ");
+            this.parentClassReference.write(writer);
+        }
+        writer.writeLine(" {");
```

Comment:
I know it's formatted anyway, but I think this should be:
```suggestion
        writer.writeLine("{");
```

Reply:
Then we can handle the additional space required in the `parentClassReference` case.